masschallenge/django-accelerator
# [AC-8824]: Update location_regional field with custom migration

[AC-8824](https://masschallenge.atlassian.net/browse/AC-8824)

How to test:
- Confirm that the migration runs on your local machine without error
- If necessary, you can run an SQL on accelerator_startup on a field `location_regional`; to ensure that Items changed are persisted
-
